Startup.cs

```diff
@@ -48,7 +48,7 @@ public void Configure(IApplicationBuilder app, IServiceProvider provider)
 
             app.UseEndpoints(endpoints => endpoints.MapControllers());
 
-            if (_options.CacheOptions.IsEnabled)
+            if (_options.CacheOptions.IsEnabled && _options.CacheOptions.ExpiresAfter > 0)
```

Comment:
I think we should allow a minimum expire time. For example if it's below 30 minutes, we throw an exception.

Reply:
I think below 10 minutes